pkg/roachprod/vm/gce/utils.go

```diff
@@ -156,6 +156,53 @@ func writeStartupScript(
 	return tmpfile.Name(), nil
 }
 
+// generateStartupScriptContent generates the startup script as a string.
+// rather than reading from a file as earlier.
+func generateStartupScriptContent(
```

Comment:
I think we should either refactor `writeStartupScript` to use this method or extract the common logic to a helper. Otherwise it seems error prone maintaining two versions of the same logic.

Reply:
wont `writeStartupScript` be removed once CLI based implementation is phased out completely. We will then be able to remove that code safely ?